### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by sydneywalcoff** [hackforla/expunge-assist#1089]

**Comment by sydneywalcoff** [comment]
@jyehllow @anitadesigns also what are your thoughts about creating a glossary page?

**Comment by sydneywalcoff** [comment]
@jyehllow I'm thinking a glossary page would be similar to the FAQ page, kind of an info dump, but with all the terms and their definitions.

The tooltips are going to be small, in-line little popups that define the terms. I'm attaching some examples below. I'm not sure what you mean in this context by footers, however.


![image](https://github.com/hackforla/expunge-assist/assets/75542938/d4bcb28e-c654-49bd-818b-09c12b7f8b55)
![image](https://github.com/hackforla/expunge-assist/assets/75542938/ee434271-3823-48fc-be5b-d0492ce1667e)
![image](https://github.com/hackforla/expunge-assist/assets/75542938/97997eb8-3395-4bb5-8d19-9851968d1dd4)

**Comment by jyehllow** [comment]
Yeah I think in terms of tooltips the first image is closest to what I was thinking. The others that darken/lighten the background and create focus on the tooltip I'm not as a big fan of, it feels super interrupting.

For footers, I actually meant footnote LOL, I know textbooks and articles usually have a thing where if there's a word that needs to be defined, they put a tag next to the word (like a number) and then define it at the bottom of the page.

![image](https://github.com/hackforla/expunge-assist/assets/75225741/fd81a183-1d69-407e-928e-cd814b3df59d)
Might be outdated at this point though, maybe tooltips are better.

**Comment by anitadesigns** [comment]
This is great to have on our radar and something for the design team to discuss further! I know Content and Research are wrapping up terminology testing they were working on last year which will help us plan next steps. Will have to ask Content as well if they would like the glossary and tool tip ready for launch so that the design team can prioritize this issue once testing is complete and Content works on #1087

**Comment by sydneywalcoff** [comment]
@anitadesigns the glossary is just an idea that I had, and would def be low priority/something to think about after this, or even after launch depending on how things shake out. 

@jyehllow sorry I thought I responded 😅 ah, footnotes, ok! I feel like footnotes might be a little academic for the ~*vibe*~ we're going for? But idk, maybe thats a discussion for the branding person? Plus we _can_ make it pretty, so why not lol.

**Comment by vanessaavviles** [comment]
Please provide update: 

1. Progress: 
2. Blockers: 
3. Availability: 
4. ETA:

**Comment by anitadesigns** [comment]
This is not ready to be worked on so I unassigned myself and justin and moved this under icebox
